handlers/handlers.go

```diff
@@ -33,3 +33,28 @@ func HandleSignUp(w http.ResponseWriter, r *http.Request) {
 
 	}
 }
+
+//HandleSignIn func
+func HandleSignIn(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	var user model.UserProfile
+	var res string
+	var err error
+
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Print(err)
+	}
+
+	err = json.Unmarshal(body, &user)
+	if err != nil {
+		log.Print(err)
+	}
+
+	err = database.LoginUser(&user)
+	if err == nil {

```

Comment:
Use same pattern if err != nil { log error and return error response}
Also, return token in json in case of no error